spanner/api/Spanner.Samples/BatchReadRecordsAsync.cs

```diff
@@ -34,7 +34,12 @@ public async Task<(int RowsRead, int Partitions)> BatchReadRecordsAsync(string p
         transaction.DisposeBehavior = DisposeBehavior.CloseResources;
         using var cmd = connection.CreateSelectCommand("SELECT SingerId, FirstName, LastName FROM Singers");
         cmd.Transaction = transaction;
-        var partitions = await cmd.GetReaderPartitionsAsync();
+
+        // A Partition object is serializable and can be used from a different process.
+        // DataBoostEnabled can be used for partition read and query to execute the request
+        // via spanner independent compute resources.
```

Comment:
I think I'd prefer to add a link here to documentation on data boost enabled instead of trying to describe that in a single sentence. If not, at least:
```suggestion
        // If data boost is enabled, partitioned read and query requests will be executed
        // using Spanner independent compute resources.
```